config/prov/prov_test.yaml

```diff
@@ -130,6 +130,8 @@ k8s_cortx_deploy:
   log_path: "/etc/cortx/log"
   custom_log_path: "/etc/cortx/custom/log"
   s3_max_start_timeout: 240
+  s3_instances_per_node: 3
```

Comment:
Should it be 1 by default ?

Reply:
in solution.example.yaml file its mentioned as 3, so kept in the same way.